ci-operator/config/maistra/api/maistra-api-maistra-2.4.yaml

```diff
@@ -0,0 +1,36 @@
+build_root:
+  image_stream_tag:
+    name: maistra-builder
+    namespace: ci
+    tag: "2.4"
+  use_build_cache: true
+resources:
+  '*':
+    limits:
+      memory: 2Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: api-unit-2-4
+  commands: |
+    make test \
+    GOBIN="/gobin" \
+    GOCACHE="/tmp/cache" \
+    GOMODCACHE="/tmp/cache" \
+    XDG_CACHE_HOME="/tmp/cache"
+  container:
+    from: src
+- as: api-gencheck-2-4
+  commands: |
+    make gen-check \
+    GOBIN="/gobin" \
+    GOCACHE="/tmp/cache" \
+    GOMODCACHE="/tmp/cache" \
+    XDG_CACHE_HOME="/tmp/cache"
+  container:
+    from: src
+zz_generated_metadata:
+  branch: maistra-2.4
```

Comment:
@bartoszmajsak This line is referring the source code repo branch.
I am checking and we haven't created branch maistra-2.4 in https://github.com/maistra/api/branches
That's why the CI job failed to pull source code.